### PR TITLE
Optimize intervals and randomize them a bit

### PR DESCRIPTION
**Pull request by Kelamir**
Related to https://github.com/koreader/koreader/issues/10028

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/10029)

### CODE REVIEW

File: plugins/vocabbuilder.koplugin/db.lua [hunk 1]
```diff
@@ -217,21 +217,19 @@ function VocabularyBuilder:gotOrForgot(item, isGot)
     local target_review_count = math.max(item.review_count + (isGot and 1 or -1), 0)
     local target_count = isGot and item.streak_count + 1 or 0
     if target_count == 0 then
-        due_time = current_time + 5 * 60
+        due_time = current_time + 5 * 60 -- 5 minutes
     elseif target_count == 1 then
-        due_time = current_time + 30 * 60
+        due_time = current_time + 24 * 3600 -- 24h
```
Comment (weijiuqiao) [inline plugins/vocabbuilder.koplugin/db.lua:222]: I'd like to retain the 30min and 12h checks to strengthen short-term memory.

Comment (Kelamir) [reply plugins/vocabbuilder.koplugin/db.lua]: Updated.

**Comment by weijiuqiao** [comment]
Looks good now.

**Comment by Kelamir** [comment]
Closing this due to a better solution being developed here: https://github.com/koreader/koreader/pull/10035

**Pull request closed without merging**